moped-etl/arcgis/components_to_agol.py

```diff
@@ -1,17 +1,20 @@
 #!/usr/bin/env python
 """Copies all Moped component records to ArcGIS Online (AGOL)"""
-# docker run -it --rm  --network host --env-file env_file -v ${PWD}:/app  moped-agol /bin/bash
+# docker compose run arcgis;
```

Comment:
I gave this a docker compose file for ease of development. I also have removed the `network=host` configuration. I can't see a reason why this needs to be available on the host's network and not tucked away in a docker environment.

Reply:
thanks for leveling up the dev environment!

Reply:
i tripped up on this too due to habit - @frankhereford could you add this to the template? i think that is a great idea because, knowing myself, i'll do it again. 🙃

Reply:
Great call y'all -- i hadn't expected a default value in there, and it makes total sense that it should be there since it's not secret. This is done.